packages/@aws-cdk/aws-apigateway/lib/domain-name.ts

```diff
@@ -22,6 +30,13 @@ export interface DomainNameOptions {
    * @default REGIONAL
    */
   readonly endpointType?: EndpointType;
+
+  /**
+   * The Transport Layer Security (TLS) version + cipher suite for this domain name.
+   * @default undefined. This field is optional in AWS::ApiGateway::DomainName SecurityPolicy
```

Comment:
What does CloudFormation default to? We should document that as the default here.

You can identify this by not setting this in CloudFormation, deploying the stack and calling the [`get-domain-name`](https://docs.aws.amazon.com/cli/latest/reference/apigateway/get-domain-name.html) from the CLI. 
I'm hoping it'll be TLS 1.2; if not, I think the CDK should pick that as the default.

Reply:
That's actually the reason I looked into this; ApiGateway/CloudFormation defaults this to TLS 1.0. I'm in favor of defaulting it to TLS 1.2, but would that be a potentially breaking change since it restricts the allowed ciphers?

Reply:
Actually, that's a very good point. We should keep it as TLS 1.0 then.